README.md

```diff
@@ -18,6 +18,8 @@ The following is an example hiera.yaml configuration for use with hiera-http
       :output: json
       :cache_timeout: 10
       :failure: graceful
+      :headers:
+        :X-Token: my-token
```

Comment:
Could you move the example code to the section below rather than here - this example is meant to show a quite minimal set of configuration thats not too confusing (see confine_to_keys) - Also - can the example not have the header name as a symbol to keep it consistent with other config...

    :headers:
      X-Token: my-token

That shouldn't need any tweaking of the code - just reads better.....
Thx